poh-bundle-linux-x64/oh.sh

```diff
@@ -1,68 +1,196 @@
+sed -e "s/3306/$MYSQL_PORT/" $POH_PATH/$OH_DIR/rsc/database.properties.sample > $POH_PATH/$OH_DIR/rsc/database.properties
+sed -e "s/MYSQL_PORT/$MYSQL_PORT/" $POH_PATH/$OH_DIR/rsc/log4j.properties.ori > $POH_PATH/$OH_DIR/rsc/log4j.properties
 
 if [ -f $POH_PATH/database.sql ]
 then
-    echo "Initializing database... on port $mysql_port"
-    cd $POH_PATH/$MYSQL_DIR/
-    rm -rf ../var/lib/mysql
-    mkdir -p ../var/lib/mysql
-    mkdir -p ../var/log/mysql
-    ./bin/mysqld --initialize-insecure --basedir=./ --datadir=../var/lib/mysql
+    echo "Initializing MySQL database... on port $MYSQL_PORT"
+#    cd $POH_PATH/$MYSQL_DIR/
+    rm -rf $POH_PATH/var/lib/mysql
+    mkdir -p $POH_PATH/var/lib/mysql
+    mkdir -p $POH_PATH/var/log/mysql
+    $POH_PATH/$MYSQL_DIR/bin/mysqld --initialize-insecure --basedir=$POH_PATH/$MYSQL_DIR --datadir=$POH_PATH/var/lib/mysql
     if [ $? -ne 0 ]; then
-		echo "Error: Initialization failed!"
+		echo "Error: MySQL initialization failed!"
 		exit 1
 	fi
 	echo "Default schemas initialized..."
-	echo -n "Starting MySQL Server."
-	./bin/mysqld_safe --defaults-file=$POH_PATH/etc/mysql/my.cnf 2>&1 > /dev/null &
-    # Wait till the MySQL socket file is created
-    while [ ! -e $POH_PATH/var/run/mysqld/mysql.sock ]; do sleep 1; done
-    ./bin/mysql --socket=$POH_PATH/var/run/mysqld/mysql.sock -u root --port=$mysql_port -e "CREATE SCHEMA oh; GRANT ALL ON oh.* TO 'isf'@'localhost' IDENTIFIED BY 'isf123'; GRANT ALL ON oh.* TO 'isf'@'%' IDENTIFIED BY 'isf123';"
-    ./bin/mysql --socket=$POH_PATH/var/run/mysqld/mysql.sock -u root --port=$mysql_port oh < $POH_PATH/database.sql
+	echo "Starting MySQL Server."
+
+	$POH_PATH/$MYSQL_DIR/bin/mysqld_safe --defaults-file=$POH_PATH/etc/mysql/my.cnf 2>&1 > /dev/null &
+
+	# Wait till the MySQL socket file is created
+	while [ ! -e $POH_PATH/var/run/mysqld/mysql.sock ]; do sleep 1; done
+
+
+	echo "Dropping OH Database (if existing)..."
+	$POH_PATH/$MYSQL_DIR/bin/mysql --socket=$POH_PATH/var/run/mysqld/mysql.sock -u root --port=$MYSQL_PORT -e "DROP DATABASE IF EXISTS $DATABASE_NAME;"
+
+	echo "Creating OH Database..."
+	$POH_PATH/$MYSQL_DIR/bin/mysql --socket=$POH_PATH/var/run/mysqld/mysql.sock -u root --port=$MYSQL_PORT -e "CREATE DATABASE $DATABASE_NAME; GRANT ALL ON oh.* TO '$DATABASE_USER'@'localhost' IDENTIFIED BY '$DATABASE_PASSWORD'; GRANT ALL ON oh.* TO '$DATABASE_USER'@'%' IDENTIFIED BY '$DATABASE_PASSWORD';"
+
+	echo "Importing schema..."
+	$POH_PATH/$MYSQL_DIR/bin/mysql --socket=$POH_PATH/var/run/mysqld/mysql.sock -u root --port=$MYSQL_PORT oh < $POH_PATH/database.sql
     if [ $? -ne 0 ]; then
-		echo "Error: Database not initialized!"
+		echo -n "Error: Database not initialized!"
 		exit 2
     fi
-    echo "Database initialized."
-    rm $POH_PATH/database.sql
+    echo "Database initialized !"
+#    rm $POH_PATH/database.sql # we don't need to remove the database file
```

Comment:
nit: also this?

Reply:
Corrected, thank you.

Reply:
well there is a reason, if I'm not mistaken, see my comment below.

Reply:
See proposed solution (renaming of the file)